tip_to_camera:
  position:
    x: -0.01585630415986354
    y: -0.01576451003413994
    z: 0.3094033757356574
  orientation:
    x: -0.4419203465619355
    y: -0.4981118042402979
    z: 0.497628197383902
    w: 0.5558391988151803
base_to_object:
  position:
    x: 0.15621997324613
    y: -0.01631437105435886
    z: -0.01150591023861538
  orientation:
    x: 0.7339648752960918
    y: 0.6791853787915489
    z: 0.0003594035631363098
    w: -0.001629078405400464
